### ISSUE TITLE
DataGrid Grouping - Flag for Expand Groups by default

### ISSUE THREAD

**Issue by DaveAPerry** [Megabit/Blazorise#5747]
I'm working on a report and want to make use of the grouping feature of the DataGrid to split out different sections in the report. When the user navigates to the page I want all the sections expanded by default and allow them to collapse the groups they aren't interested in.

**Describe the solution you'd like**
I think a parameter on the DataGrid for "GroupCollapse" (or something like that) that could be set to Collapse or Expand to configure the default state.

**Comment by David-Moreira** [comment]
Hello @DaveAPerry 
While a flag might be helpful we do have a programatic way of triggering the expand/collapse operations.
Have you tried the `ExpandAllGroups`? When called at the appropriate time it should do as you ask.

**Comment by DaveAPerry** [comment]
@David-Moreira Yes, I have tried using Expand all groups, and it does work if I can get it to run at the proper time - I can get it to run on the initial render well enough - I need to put some more time in to get it to work properly when the component is rerendered (ie. Switching between blazorise tabs).

It's more of a nice to have idea than a need to have idea.

**Comment by holderbr** [comment]
> I was able to get it working - you can close this issue if you want.
> 
> Again, it's a nice to have not a need to have.

I know this is old, but do you mind sharing your approach?  I've been trying to get this working, without consistent results.

**Comment by DaveAPerry** [comment]
> > I was able to get it working - you can close this issue if you want.
> > Again, it's a nice to have not a need to have.
> 
> I know this is old, but do you mind sharing your approach? I've been trying to get this working, without consistent results.

Sorry, I don't remember and am no longer working on that project to be able to check. I believe it was calling the ExpandAllGroups after the component was rendered.
